doc/release-notes.md

```diff
@@ -63,6 +63,15 @@ P2P and network changes
 
 Updated RPCs
 ------------
+- RPCs `getmempoolentry`, `getrawmempool verbose=true`, `getmempoolancestors
+  verbose=true` and `getmempooldescendants verbose=true` no longer return the
+  top-level fee fields `fee`, `modifiedfee`, `ancestorfees` and
+  `descendantfees`. These fields were deprecated in v0.17 and moved to the
+  `fees` JSON object in the response. The `-deprecated=fees` configuration
+  option must be passed for these fields to be included in the top-level
+  response. This option will only be available in this major version, after
+  which the fields will be fully removed. Use the `fees` JSON object for these
+  RPCS instead. (#22689)
```

Comment:
Mostly wording suggestions, but also I think you should warn users about the fact that the fees are denominated slightly differently when they switch to using the `fees` object.
```suggestion
- The top-level fee fields `fee`, `modifiedfee`, `ancestorfees` and `descendantfees`
  returned by RPCs `getmempoolentry`,`getrawmempool(verbose=true)`,
  `getmempoolancestors(verbose=true)` and `getmempooldescendants(verbose=true)`
  are now deprecated and will be removed in the next major version (use
  `-deprecated=fees` if needed in this version). The same values, denominated
  in satoshis, can be accessed through the `fees` object in the result. (#22689)
```

Reply:
Doesn't seem to be an issue here, but for future reference, you might want to put it in a release-notes-22689.md to avoid conflicts with other PRs updating release notes.

Reply:
> slightly differently when they switch to using the fees object

just double checking, the update should read "denominated in BTC," correct?

Reply:
yes, BTC, sorry